lib/testsuite/context.js

```diff
@@ -317,7 +317,17 @@ class Context extends EventEmitter {
     }
 
     if (this.allScreenedTests.includes(testName)) {
-      throw new Error(`A testcase with name "${testName}" already exists. Testcases must have unique names inside a test suite.`);
+      const {Logger} = Utils;
+
+      const err = new Error(
+        'An error occurred while loading the testsuite:\n' +
+        `A testcase with name "${testName}" already exists. Testcases must have unique names inside the test suite, ` +
+        'otherwise testcases with duplicate names might not run at all.\n\n' +
+        'This testsuite has been disabled, please fix the error to run it properly.'
+      );
+      Logger.error(err);
+
+      this.setAttribute('@disabled', true);
```

Comment:
This acts more like a warning than throwing the error and exiting the process, while also making sure the faulty testsuite is disabled and the user sees it in the report.

![image](https://github.com/user-attachments/assets/77c3cdb8-b6fd-497c-b745-525d4070076e)